docs/book/src/quick-start.md

```diff
@@ -9,7 +9,7 @@ This Quick Start guide will cover:
 
 ## Prerequisites
 
-- [go](https://golang.org/dl/) version v1.15+ and < 1.16.
+- [go](https://golang.org/dl/) version v1.15+.
```

Comment:
That need to be 1.16 now. Could you please fix that? Also, could you please check if has any other place as well? I think we have a code check related to the go versions either? Could you give to us a hand on that?

Reply:
c/c @rashmigottipati